static/js/components/EducationDialog.js

```diff
@@ -8,21 +9,22 @@ import ProfileFormFields from '../util/ProfileFormFields';
 import { educationValidation } from '../util/validation';
 
 export default class EducationDialog extends ProfileFormFields {
-  constructor(props) {
+  constructor(props: {}) {
```

Comment:
I think I prefer `props: Object` from a readability point of view. Is there any difference to flow?